projects/js-packages/social-logos/tools/svg-optimize.js

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env node
+/* eslint-disable no-console */
+const srcSvgDir = 'src/svg';
+const destSvgDir = 'build/svg-clean';
+
+const fs = require( 'fs' );
+const path = require( 'path' );
+const process = require( 'process' );
+const { glob } = require( 'glob' );
+const { optimize } = require( 'svgo' );
+
+const svgo_config = {
+	js2svg: { finalNewline: true }, // force EOF newline
+	plugins: [
+		{
+			name: 'preset-default',
+			params: {
+				overrides: {},
+			},
+		},
+		{
+			name: 'removeAttrs',
+			params: {
+				attrs: [ 'style', 'xml:space', 'id', 'fill' ],
+				elemSeparator: '!',
+			},
+		},
+		{
+			name: 'addAttributesToSVGElement',
+			params: {
+				attributes: [ { viewBox: '0 0 24 24' }, { xmlns: 'http://www.w3.org/2000/svg' } ],
+			},
+		},
+	],
+};
+
+// Start in the right folder.
+const rootDir = __dirname + '/..';
+process.chdir( rootDir );
+
+// Make dir if it doesn't exist.
+if ( ! fs.existsSync( destSvgDir ) ) {
+	fs.mkdirSync( destSvgDir, { recursive: true } );
+}
```

Comment:
With `recursive: true`, it does the right thing if the dir already exists. So we can just do
```suggestion
fs.mkdirSync( destSvgDir, { recursive: true } );
```

Reply:
dbf5a12c9f